index.js

```diff
@@ -7,14 +7,15 @@ const axios = require('axios');
  * Returns the networth of a profile
  * @param {object} profileData - The profile player data from the Hypixel API (profile.members[uuid])
  * @param {number} bankBalance - The player's bank balance from the Hypixel API (profile.banking?.balance)
+ * @param {object} museumData - The player's museum data from the Hypixel API (museum.members[uuid])
  * @param {{ cache: boolean, onlyNetworth: boolean, prices: object, returnItemData: boolean }} options - (Optional) cache: By default true (5 minute cache), if set to false it will always make a request to get the latest prices from github, onlyNetworth: If true, only the networth will be returned, prices: A prices object generated from the getPrices function. If not provided, the prices will be retrieved every time the function is called, returnItemData: If true, the item data will be returned in the object
  * @returns An object containing the player's networth calculation
  */
-const getNetworth = async (profileData, bankBalance, options) => {
+const getNetworth = async (profileData, bankBalance, museumData, options) => {
```

Comment:
will break everyone who is updating and using options, some other options to include museumData idk. Also conflicts with the example